config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -149,16 +177,13 @@ presubmits:
       containers:
       - image: launcher.gcr.io/google/bazel:0.24.1
         command:
-        - bash
+        - ../test-infra/hack/bazel.sh
         args:
-        - -c
-        - |
-              code=0
-              if ! bazel test --config=unit --config=remote --remote_instance_name=projects/fejta-prod/instances/default_instance //... -- -//build/... -//vendor/...; then
-                code=1
-              fi
-              ../test-infra/hack/coalesce.py || true
-              exit "$code"
+        - test
+        - --config=unit
+        - --config=remote
+        - --remote_instance_name=projects/k8s-prow-builds/instances/default_instance
+        - //...
```

Comment:
Did this intentionally drop `-//build/... -//vendor/...`?

Reply:
No, I'm just :see_no_evil: